Changelog.md

```diff
@@ -1,6 +1,57 @@
+Changed|**GET** `/cases/{case_id}`|Added new querystring parameter include_fields, case_secret and external_user_unique_key
+Changed|**PUT** `/cases/{case_id}/fields`|Added new querystring parameter external_user_unique_key
+Added|**POST** `/cases/{case_id}/reporter_form_email_requests`|Creates a request to e-mail the reporter form for the given case
+Added|**GET** `/cases/{case_id}/reporter_form`|Returns if the reporter form PDF is available
+Added|**GET** `/cases/{case_id}/reporter_form/download`|Download the reporter form PDF file
+Added|**POST** `/cases/{case_id}/state_change_requests`|Request a state change
+Added|**POST** `/scheduled_tasks/{scheduled_task_id}/cases`|Create a new case from a scheduled task
+Added|**GET** `/cases/{case_id}/case_forms`|Gets all case_forms for a case
+Added|**GET** `/cases/{case_id}/case_forms/{case_form_id}`|Gets a specific case_form
+Added|**PATCH** `/cases/{case_id}/case_forms/{case_form_id}`|Updates a case_form
+Changed|**GET** `/cases/case_types/{case_type_id}`|Added new querystring parameter include_states, include_subject_tree and external_user_unique_key
+Added|**GET** `/cases/case_types/{case_type_id}/states`|Get states of case type
+Changed|**GET** `/cases/case_types/{case_type_id}/permissions`|Added new querystring parameter assigned_to_me, include_assigned_to_me, include_teams, include_user_groups and include_users
+Changed|**GET** `/portals/{portal_id}/collections`|Added new querystring parameter include_sub_tree
+Added|**GET** `/cases/coordinator_forms/{form_id}/design`|Gets the design of a coordinator form
+Added|**GET** `/cases/coordinator_forms/{form_id}`|Gets a coordinator form for an id
+Added|**GET** `/cases/coordinator_forms/{form_id}/images/{image_id}/download`|Gets the binary image file of a coordinator form
+Added|**GET** `/external_sources`|Get external sources
+Added|**GET** `/external_sources/{external_source_id}`|Get a single external source
+Added|**GET** `/external_sources/{external_source_id}/data`|Get the data of the external source
+Added|**GET** `/external_sources/{external_source_id}/data/{filter_id}`|Get the data of the external source using a filter
+Added|**POST** `/external_sources/{external_source_id}/data/filter`|Creates a external source data filter
+Added|**GET** `/frameworks`|Get frameworks
+Added|**GET** `/cases/preview_forms/{preview_form_id}/design`|Gets the design of a preview form
+Added|**GET** `/cases/preview_forms/{preview_form_id}`|Gets a preview form for an id
+Added|**GET** `/cases/preview_forms/{form_id}/images/{image_id}/download`|Gets the binary image file of a preview form
+Changed|**GET** `/cases/reporter_forms`|Added new querystring parameter can_report
+Added|**GET** `/cases/reporter_forms/{form_id}/images/{image_id}/download`|Gets the binary image file of a form
+Added|**GET** `/saml/saml2`|Retrieves the url to call to authenticate using ADFS with parameters forceauthn, redirect_url, for_token, for_bearer_token and use_current_ip_context
```

Comment:
volgens mij is deze route niet nieuw, maar was deze wel dubbel met die saml en saml/adfs routes

Reply:
Klopt de vergelijking was wat vreemd, heb het aangepast.